File: task_planner/src/main/java/com/jurgen/task_planner/models/dtos/TaskDto.java
package com.jurgen.task_planner.models.dtos;

import java.time.OffsetDateTime;
import java.util.List;

public record TaskDto(
    long id,
    long issueId,
    String code,
    String description,
    List<TaskAssigneeDto> assignees,
    OffsetDateTime createdAt,
    OffsetDateTime updatedAt
) {}

File: react_fe/src/models/dto/TaskDto.ts
import { TaskAssigneeDto } from './TaskAssigneeDto'

export interface TaskDto {
  id: number
  issueId: number
  statusId: number
  statusName: string
  code: string
  description: string
  assignees: TaskAssigneeDto[]
  createdAt: string
  updatedAt: string
}
